app/src/main/java/org/fossasia/openevent/activities/SpeakersActivity.java

```diff
@@ -55,13 +55,16 @@
 
     private boolean flagSocial = false;
 
+    private TextView noSessionsView;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'noSessionsView' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406630500/issues/source?bid=3233850&fileBranchId=3233897#l58)